specification/migrate/resource-manager/readme.md

```diff
@@ -20,6 +20,21 @@ batch:
   - tag: package-migrateengine-2022-05
 ```
 
+### Tag: package-preview-2023-09
+These settings apply only when `--tag=package-preview-2023-09` is specified on the command line.
+
+```yaml $(tag) == 'package-preview-2023-09'
+input-file:
+  - Microsoft.Migrate/AssessmentProjects/preview/2023-09-09-preview/migrate.json
+suppressions:
+  - code: AvoidAdditionalProperties
+    reason: Migrate feature is widely adopted and requires additionalProperties for these swagger properties.
+  - code: ProvisioningStateMustBeReadOnly
+    reason: The current swagger version only modifies the resources which do not have Provisioning states as readOnly and not introducing any new Resources, These are incorrectly flagged for previous versions.
+  - code: UnSupportedPatchProperties
+    reason: There is no patch operation that is introduced in this swagger version where it is not readOnly. This is incorrectly flagged for previous versions.
```

Comment:
>There is no patch operation that is introduced in this swagger version [](http://example.com/codeflow?start=12&length=70)

there are new properties added to this pr that could be unsupported by existing PATCH operations if the newly added properties are included in an object used in PATCH. Which property the linter is complining about on this pr ?

Reply:
you should add the where clause for the suppresions so that the reviwers can see them to do a suppresion approval.

Reply:
i do not see a provisioningState added by this PR. is this suppression needed also ?

Reply:
"provisioningState" is an existing property and Lint was flagging it. Hence added the suppression.
As mentioned above, these changes are also going to get merged with 2023-05-01-preview version. Once done, I will be removing these changes from my PR.
It was just added to temporarily fix the lint errors.